index.js

```diff
@@ -36,13 +57,18 @@
   // Text: 'The man who runs may fight again.\n\t\t-- Menander'
   // Hint: This is going to require setting the content's position to relative first
   // Hint: Additionally, I styled the paragraph element's width, text alignment, top and left positions
-
+const txt = document.createElement('p');
+txt.style.position = "relative";
+// console.log(fortune['fortune']);
+house.appendChild(txt);
+txt.style.right = "350px";
+txt.style.bottom = "15px";
 // Make an Object (a.k.a a 'dictionary') called `fortune` and add one key-value pair:
-  // 'fortune' : 'The man who runs may fight again.\n\t\t-- Menander'
-
+// 'fortune' : 'The man who runs may fight again.\n\t\t-- Menander'
+const fortune = {'fortune' : 'The man who runs may fight again.\n\t\t <br\> -- Menander'}
 // Cool, now just trust me for this next one...
-  // Set the inner HMTL of the paragraph element to the value of the fortune key in the fortune object
-  
+// Set the inner HMTL of the paragraph element to the value of the fortune key in the fortune object
+txt.innerHTML += fortune['fortune'];
```

Comment:
This'll work, just remember that it's "adding" a new fortune in addition to the existing one.